saylua/modules/pets/models/db.py

```diff
@@ -1,38 +1,57 @@
-from google.appengine.ext import ndb
+from saylua import db
 
 
 # This is to store alternate linart versions of the same pets
-class SpeciesVersion(ndb.Model):
-    name = ndb.StringProperty()
-    base_image = ndb.StringProperty()
-    base_psd = ndb.StringProperty()
-    default_image = ndb.StringProperty()
+class SpeciesVersion(db.Model):
```

Comment:
Upon further though I think it'd be best to start off not worrying about alternate linearts.